packages/sit-onyx/src/components/OnyxAccordion/OnyxAccordion.vue

```diff
@@ -31,10 +32,16 @@ const updateOpen = (id: string, isOpen: boolean) => {
     openItems.value.delete(id);
     return;
   }
-  if (props.exclusive) openItems.value.clear();
+  if (exclusive.value) openItems.value.clear();
   openItems.value.add(id);
 };
 
+watch(exclusive, (newExclusive) => {
+  if (newExclusive) {
+    openItems.value.clear();
```

Comment:
We only need to clear if more than 1 accordion is open. If exactly one is open thats fine.
Did you discuss with Jannick how this should behave? Maybe he decides that we don't want to handle this at all